src/js/plugin.js

```diff
@@ -312,6 +312,9 @@ class Plugin {
     if (name !== BASE_PLUGIN_NAME) {
       if (Plugin.isBasic(plugin)) {
         Player.prototype[name] = createBasicPlugin(name, plugin);
+        Object.keys(plugin).forEach(function(prop) {
+          Player.prototype[name][prop] = plugin[prop];
+        });
```

Comment:
If we really want it in createBasicPlugin, then we have to do the `Player.prototype[name]` assign in there as well. I don't see any problems with that. Thoughts?